client/sidebar/sections/Omnichannel.tsx

```diff
@@ -18,18 +21,20 @@ const OmnichannelSection = (props: typeof Box): ReactElement => {
 	const dispatchToastMessage = useToastMessageDispatch();
 
 	const changeAgentStatus = useMethod('livechat:changeLivechatStatus');
-	const [voipCallAvailable, setVoipCallAvailable] = useState(useOmnichannelVoipCallAvailable());
-
+	const voipCallAvailable = useState(useOmnichannelVoipCallAvailable());
+	const [registered, setRegistered] = useState(false);
+	const voipLibIsReady = useIsVoipLibReady();
+	const voipLib = useVoipUser();
 	const agentAvailable = useOmnichannelAgentAvailable();
 
 	const showOmnichannelQueueLink = useOmnichannelShowQueueLink();
 	const queueLink = useOmnichannelQueueLink();
 	const directoryLink = useOmnichannelDirectoryLink();
 
 	const voipCallIcon = {
-		title: voipCallAvailable ? t('Available') : t('Not_Available'),
-		color: voipCallAvailable ? 'success' : undefined,
-		icon: voipCallAvailable ? 'phone' : 'phone-disabled',
+		title: !registered ? t('Enable') : t('Disable'),
+		color: !registered ? 'success' : undefined,
```

Comment:
@amolghode1981 , can we change the direction of the condition here? 😬

Reply:
@KevLehman  @MartinSchoeler  please help me understand this. 
There is icon  as "icon: !registered ? 'phone' : 'phone-disabled',"

1. Title : What does title signify? Current state or action on click? If it is an action, Enable makes sense when the endpoint is not registered.
2. color : Same as above, if it is action on click, then after registering the colour should be green but then then phone icon turns out to be phone-disabled. 
I am confused about it.

Reply:
If the sidebar is using buttons or action buttons it should go like that 
Title: Is what will show up on the tooltip when you hover it

The color should be
`		color: registered ? 'success' : undefined,`
Since if the user is not registered, the button should not be green, just like the livechat button, when is not enabled, it is grey.

Reply:
Fixed.